src/Layout/Navigation/RecentItem.tsx

```diff
@@ -11,26 +11,137 @@
  */
 
 import React from 'react';
-import { Link } from 'react-router-dom';
-import { NavItem } from '@patternfly/react-core';
+import { AlertVariant, Dropdown, DropdownItem, KebabToggle, NavItem } from '@patternfly/react-core';
 import { NavigationRecentItemObject } from '.';
-import isActive from './isActive';
+import getActivity from './isActive';
 import WorkspaceIndicator from '../../components/Workspace/Indicator';
+import WorkspaceActionsProvider from '../../containers/WorkspaceActions';
```

Comment:
I don't have a strong opinion, but what I proposed and we kind of agree on UX call: user should be redirected to Workspace Loading page and they `Restart the current workspace`, now we should keep offline IDE, that is not refreshed after workspace is restarted:
![Screenshot_20210402_094609](https://user-images.githubusercontent.com/5887312/113389306-7d36fd80-9398-11eb-85fe-a55d5e3c17bd.png)
![Screenshot_20210402_094743](https://user-images.githubusercontent.com/5887312/113389308-7dcf9400-9398-11eb-93ea-d50dfd53b2c2.png)

Reply:
Also, what was proposed @benoitf to avoid confusion if Restart is happen in background or not: 
`Restart workspace` action should behave differently if:
- IDE of the chosen workspace is displayed - we redirect user to Loading page;
- Workspace loading page of the chosen workspace is displayed - we redraw the current state of the workspace (Stopping/Starting);
- Any other tab(IDE of another workspace, workspace details, Container registries) - we restart a workspace in background.